openid-4-verifiable-presentations-1_0.md

```diff
@@ -611,6 +629,20 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+## Attachments in Authorization Requests {#attachments-in-authorization-requests}
+
+The attachment data mechanism enables to pass additional information to the Wallet to request credentials. The Verifier MAY include attachments in the Authorization Request, for example to pass a registration certificate to inform about the registered intended use or an authorization attestation to request a specific credential from the Wallet to comply with the embedded disclosure policy.
```

Comment:
this needs more explanation, right now it is too much EUDIW lingo and does not describe the broader context.

Reply:
I am open for other use cases, but I think the main points is for authentication/authorization.

In most cases authorization

Reply:
updated